pytket/pytket/_tket/passes.pyi

```diff
@@ -54,7 +54,7 @@ class BasePass:
         :param after_apply: Invoked after a pass is applied. The CompilationUnit and a summary of the pass configuration are passed into the callback.
         :return: True if pass modified the circuit, else False
         """
-    def to_dict(self) -> dict:
+    def to_dict(self) -> json:
```

Comment:
Why can't this still return a `dict`?

Reply:
This is auto generated based on the `serialise` method now returning `json` in the c++